### ISSUE TITLE
Focus in sidenav panel only goes between "Learn" and the close button (Firefox)

### ISSUE THREAD

**Issue by sairina** [learningequality/kolibri#9189]
## Observed behavior
When the sidenav panel is opened and keyboard navigation is used, the focus only goes between "Learn" and close button.
![2022-03-14 12 03 16](https://user-images.githubusercontent.com/13563002/158268305-b3f847fa-6ff7-4fff-9fca-29a0a22b27ac.gif)

## Errors and logs
None

## Expected behavior
User should be able to access all links via keyboard

## User-facing consequences
User cannot access all links via keyboard

## Steps to reproduce
1. Sign in
2. Navigate with the keyboard to the side nav (hamburger menu)
3. Open the side nav, and continue to use keyboard for navigation

## Context
Kolibri 0.15
Firefox

**Comment by marcellamaki** [comment]
Possible mac settings issue 
![main-side-nav-firefox.gif](https://images.zenhubusercontent.com/6047f6e691103a78d3ad0ace/4db67d62-ef5a-48cb-a1c3-4703fa8385a0)

**Comment by marcellamaki** [comment]
cc @radinamatic -- to double check before closing

**Comment by pcenov** [comment]
With the correct settings enabled the sidebar navigation works correctly on Safari, Firefox and Chrome on Mac.
